src/index.html

```diff
@@ -1,22 +1,385 @@
+                <h3 class="recommended__title">BEOSOUND HX</h3>
+                <p class="recommended__description">
+                  Wireless noise-cancelling headphones. Listen in comfort
+                  whether you're on a call, working from home or taking a moment
+                  for yourself.
+                </p>
+                <p class="recommended__price">$ 499.00</p>
+              </article>
+              <article class="recommended__product">
+                <img
+                  src="image/products/BEOSOUND-THEATRE.png"
+                  alt="BEOSOUND THEATRE"
+                  class="recommended__photo"
+                />
+                <h3 class="recommended__title">BEOSOUND THEATRE</h3>
+                <p class="recommended__description">
+                  Four patent-pending acoustic innovations in one sleek soundbar
+                  envelop you in sound from every angle. Made to fit any TV, and
+                  last for generations.
+                </p>
+                <p class="recommended__price">$ 3199.00</p>
+              </article>
+            </div>
+          </section>
+          <section
+            class="categories"
+            id="categories"
+          >
+            <h2 class="section-title">
+              Sound. Vision.
+              <br />
+              It&#39;s your pick.
+            </h2>
+            <div class="categories__wrappe">
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/portable-speakers-first.png"
+                      alt="portable-speakers"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/portable-speakers-second.png"
+                      alt="portable-speakers"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+                <h3 class="category__title">Portable speakers</h3>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/earphones-first.png"
+                      alt="earphones"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="image/categories/earphones-second.png"
```

Comment:
You are using small images, that provides a very bad quality when using it. Try to copy these files correctly from figma, downloading the best quality image to be used. this will provide better interface for the user:

<img width="1704" alt="Screenshot 2025-01-14 at 09 54 29" src="https://github.com/user-attachments/assets/99400b79-0fc2-45e4-9980-5d25a13b6b85" />